pkg/kv/kvserver/deleted_external_sstable_test.go

```diff
@@ -223,6 +257,15 @@ func TestGeneralOperationsWorkAsExpectedOnDeletedExternalSST(t *testing.T) {
 					deleteRangeHelper(ctx, store, roachpb.Key("a"), roachpb.Key("b")))
 				require.NoError(t,
 					deleteRangeHelper(ctx, store, roachpb.Key("g"), roachpb.Key("h")))
+
```

Comment:
The test is fairly repetitive, maybe this could be condensed?

- have a list of "things to try" in each scenario
- each test case specifies the result of each

when something about the "things to try" changes, only have to add it in one place?

Maybe this isn't a flexible enough format, but I think boiling this down somewhat is a good idea.